packages/document-renderer/src/index.tsx

```diff
@@ -42,6 +42,7 @@ interface Renderers {
     blockquote: OnlyChildrenComponent;
     code: Component<{ children: string }> | keyof JSX.IntrinsicElements;
     layout: Component<{ layout: [number, ...number[]]; children: ReactElement[] }>;
+    layoutArea: OnlyChildrenComponent;
```

Comment:
Note I've changed this so the layout renders the `div`s rather than add another renderer to be consistent with how the lists work and so that e.g. if you wanted to customize the rendering of the layout area based on the index that the layout area is at, you can easily do that because you're controlling the rendering of both the layout and the layout area in one place

Reply:
@mitchellhamilton but won't that make it inconsistent with the model slate provides? I didn't invent the `layout-area` key 🤔 
![image](https://user-images.githubusercontent.com/22598347/142376342-caf6b40d-51a8-40d8-b1e0-1f81d6f7214c.png)

this essentially blocks the ability to provide a custom renderer for layout-areas

Reply:
Not sure how you mean it's inconsistent with the model slate provides, The layout-area nodes still exist, just because there's a node of a given type doesn't mean there has to a one to one mapping of node type to renderer.

The way you provide a custom renderer for layout areas is by providing the custom renderer for layout and mapping over children and rendering whatever you want for each layout area, that way you have more control over rendering like you could decide to render the first layout area differently than all the other ones or anything else like that because you control the mapping of the layout areas. This is the same way the list renderer works, there's no list item renderer, only list.